docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Reduce storage for Bonsai Tries
+sidebar_position: 12
+description: Enable this feature to reduce the size of your database when using Bonsai Tries
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+To decrease the database size when using the [Bonsai Trie](../concepts/data-storage-formats#bonsai-tries) data storage format, enable the early access feature `--Xbonsai-limit-trie-logs-enabled`. 
+When enabled, this feature can reduce database growth by more than 3GB each week on mainnet.
+
+## Limit trie logs
+
+:::caution
+
+The following commands are examples. Before executing these example commands on your node, modify them to apply to your node's configuration.
+
+:::
+
+1. Add the `--Xbonsai-limit-trie-logs-enabled` option to the [Besu configuration file](use-configuration-file).
+   
+    :::note
+    
+    If you are using a `systemd` service file, as recommended by [CoinCashew](https://www.coincashew.com/coins/overview-eth/guide-or-how-to-setup-a-validator-on-eth2-mainnet/part-i-installation/step-3-installing-execution-client/besu) 
+    and [Somer](https://someresat.medium.com/guide-to-staking-on-ethereum-ubuntu-teku-f09ecd9ef2ee), ensure you execute `sudo systemctl daemon-reload`.
+    
+    :::
+   
+1. Stop Besu.
+1. (Optional) Run the Besu trie log prune command. Specify the Bonsai Trie data storage format and the data directory for your Besu database:
```

Comment:
```suggestion
1. (Optional) [Prune outdated trie logs.](#prune-outdated-trie-logs) Specify the Bonsai Trie data storage format and the data directory for your Besu database:
```

I assume the "Prune outdated trie logs" section goes into detail about this step. If so, it should be referenced here.

Reply:
I think we just want to provide the command here. Sending them to the Prune section will give them information in a circuitous way.